smart_card_connector_app/src/pcsc-api-jstocxxtest.js

```diff
@@ -1072,6 +1072,130 @@ goog.exportSymbol('testPcscApi', {
       assert(called);
       assertEquals(result.getErrorCode(), API.SCARD_S_SUCCESS);
     },
+
+    // Test `SCardGetAttrib()` succeeds and returns information about the card.
+    'testSCardGetAttrib_successAtr': async function() {
+      await launchPcscServer(
+          /*initialDevices=*/[{
+            'id': 123,
+            'type': SimulationConstants.GEMALTO_DEVICE_TYPE,
+            'cardType': SimulationConstants.COSMO_CARD_TYPE
+          }]);
+      const context = await establishContextOrThrow();
+      const cardHandle = await connectToCardOrThrow(
```

Comment:
from `Connect` doc: `Establishes a connection to the reader specified in * szReader.`. Maybe this function should be called `connectToReader` then(instead of card)?

Reply:
Fair point; renamed.